README.md

```diff
@@ -175,8 +175,6 @@ This involves creating an external credential file and using it as `--secret-fil
 
 #### Option 3: Using GKE Workload Identity
 
-Keep in mind that [Workforce Identity Federation Users cannot generate signed URLs](https://cloud.google.com/iam/docs/federated-identity-supported-services#:~:text=workforce%20identity%20federation%20users%20cannot%20generate%20signed%20URLs.). This means, if you are using Workforce Identity Federation, you will not be able to run `velero backup logs`, `velero backup download`, `velero backup describe` and `velero restore describe`.
```

Comment:
Per the conversation in issue https://github.com/vmware-tanzu/velero/issues/6921, I think this information is still true for most Workload Identity scenarios, especially for the programmatic way, so I think we still need to keep this line, and we just need to add a note for the exception scenario.

Reply:
it's workload identities, typical use case of trust relationship between k8s SA and IAM SA and they can indeed create signed URLs etc. . With workload identities we could always do backups, snapshots, describe, download, logs, restore, everything you expect to be able to do with Velero.

We have been using workload identities with Velero since v1.5.2 and v1.1.0 for the GCP plugin and on multiple clusters and never had an issue up until now with the regression due to workforce support.